Dockerfile

```
@@ -0,0 +1,28 @@
+FROM ubuntu:22.04 
+
+LABEL description="Build environment for Miyoo Custom Firmware" 
+
+COPY arm-miyoo-linux-uclibcgnueabi_sdk-buildroot.tar.gz /opt/miyoo
```

We need the extracted files in the image - fortunately we already extracted them right after downloading.

```suggestion
COPY arm-miyoo-linux-uclibcgnueabi_sdk-buildroot /opt/miyoo
```